tsconfig.json

```diff
@@ -1,21 +1,17 @@
 {
   "compilerOptions": {
+    "declaration": true,
     "allowSyntheticDefaultImports": true,
     "module": "commonjs",
     "removeComments": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "noUnusedLocals": true,
     "moduleResolution": "node",
-    "target": "es2017",
-    "lib": ["es2017", "dom", "esnext.asynciterable"],
-    "sourceMap": true,
-    "outDir": "./dist",
-    "baseUrl": "./"
+    "target": "ES6",
```

Comment:
Why this? ES2017 is newer and has nice features
https://medium.com/komenco/what-to-expect-from-javascript-es2017-the-async-edition-618e28819711

Reply:
will change it